sources/packages/api/scripts/setupDB.ts

```diff
@@ -1,29 +1,23 @@
-require("../env_setup");
-import { createConnection } from "typeorm";
-const config = require("../ormconfig");
+import { ormConfig, simsDataSource } from "../src/database/config/ormconfig";
 
 /**
  * Script main execution method
  */
 (async () => {
   try {
-    // Create Connection
+    // Create DataSource
     console.log("**** Running setupDB ****");
-    delete config.entities;
-    const connection = await createConnection({
-      ...config,
-      logging: ["error", "warn", "info"],
-    });
-    await connection.query(`CREATE SCHEMA IF NOT EXISTS ${config.schema};`);
-    await connection.query(`SET search_path TO ${config.schema}, public;`);
-    await connection.query(`SET SCHEMA '${config.schema}';`);
+    const connection = await simsDataSource.initialize();
```

Comment:
simsDataSource.initialize() return the datasource instance not the connection. 
Hence the variable can be renamed to datasource instead of connection.

![image](https://user-images.githubusercontent.com/54600590/180049263-fed1c732-830f-466d-b5c6-ab056d1918e4.png)